resources/web/wwi/Makefile

```diff
@@ -32,11 +32,30 @@ JS_SOURCES = \
   x3d.js \
   x3d_scene_manager.js \
   animation.js \
-  webots.js
+  webots.js \
+  node_modules/@babel/polyfill/dist/polyfill.js
 
-release debug profile update: $(JS_SOURCES)
-	@echo "# Compressing webots.js.min"
-	@terser $(JS_SOURCES) -o webots.min.js
+NPM_EXISTS = $(shell which npm 2> /dev/null)
+
+ifeq ($(NPM_EXISTS),)
+release debug profile update:
+	@echo "# Please install nodejs to build 'webots.min.js' using the instructions on the GitHub wiki."
+else
+release debug profile update: install $(JS_SOURCES)
+	@echo "# compressing webots.min.js"
+	@node_modules/.bin/babel $(JS_SOURCES) -o webots.max.js --presets "@babel/preset-env"
+	@node_modules/.bin/minify webots.max.js -o webots.min.js
+	@rm webots.max.js
+endif
+
+install:
```

Comment:
I believe one rule is missing here, as the install takes place every time I type make (and takes a few seconds to complete on Windows), which shouldn't be the case.

Reply:
I think I found a solution to this issue using a temporary file (named install.token) which determines when the build occured.

As a proof of success, please examine the following command serie:

```shell
$ make uninstall && echo "Make 1" && make && echo "Make 2" && make && echo "Make 3" && touch package.json && make
# uninstalling npm dependencies
Make 1
# uninstalling npm dependencies
# installing npm dependencies
# compressing webots.min.js
Make 2
# compressing webots.min.js
Make 3
# uninstalling npm dependencies
# installing npm dependencies
# compressing webots.min.js
```

Reply:
Make 2 should not recompile `webots.min.js`. I am fixing this and will commit something soon.